cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -454,6 +472,43 @@ func generateMountsCommonInstallRecover(mst *initramfsMountsState) error {
 	return sysconfig.ConfigureTargetSystem(configOpts)
 }
 
+func maybeMountSave(disk disks.Disk, rootdir string, encrypted bool, mountOpts *systemdMountOptions) (haveSave bool, err error) {
```

Comment:
thanks for adjusting this logic, this function makes much more sense to me now and matches my new understanding of the ubuntu-save feature scope